src/charts/trends/trends-chart.html

```diff
@@ -19,7 +19,7 @@
   </div>
   <div ng-switch-when="compact" class="trend-card-compact-pf">
     <div class="row trend-row">
-      <div class="col-sm-4 col-md-4">
+      <div class="col-sm-4 col-md-4" ng-class="{'col-sm-push-8 col-md-push-8': $ctrl.config.compactLabelPosition === 'right'}">
```

Comment:
@amarie401 just a callout... as discovered this is going to conflict with @dtaylor113 PR...